src/results-output/results-doc.ts

```diff
@@ -102,18 +103,17 @@ export function setSession(session: NReplSession, newNs: string): void {
     }
 }
 
-export function isResultsDoc(doc: vscode.TextDocument): boolean {
-    return doc && path.basename(doc.fileName) === RESULTS_DOC_NAME;
+export function isResultsDoc(doc?: vscode.TextDocument): boolean {
+    return !!doc && path.basename(doc.fileName) === RESULTS_DOC_NAME;
```

Comment:
What's the difference in using double negation here vs just `doc && ...`?

Reply:
with `doc && ...` if `doc` was `undefined` then this function would return `undefined`. the function type says it returns a `boolean`, though, and while we could make it `boolean | undefined` I prefer to keep the types more constrained and simpler if I can

Reply:
Ah, thanks!